scripts/utils/generateMain.js

```diff
@@ -177,11 +181,23 @@ function toPascalCase (str) {
   return str[0].toUpperCase() + str.slice(1)
 }
 
-function buildMethodDefinition (api, name, hasBody) {
+function buildMethodDefinition (opts, api, name, hasBody) {
   const Name = toPascalCase(name)
   const bodyType = ndjsonApiKey.includes(Name) ? 'RequestNDBody' : 'RequestBody'
   const defaultBodyType = ndjsonApiKey.includes(Name) ? 'Record<string, any>[]' : 'Record<string, any>'
 
+  if (opts.kibana) {
```

Comment:
It looks like the script requires at least some minimal test-set. Do you need any help with this?

Reply:
The generated code is tested with the integration test and the types test.
The types test will now fail because the kibana output exposes less properties, which is expected.
I’m not entirely sure if it makes sense to write a test specifically for the script.

Reply:
> The generated code is tested with the integration test and the types test. 
> The types test will now fail because the kibana output exposes less properties, which is expected.

This is good as long as Kibana-specific types are covered. Can we add a separate test for them?